src/ui/MissingTerm/MissingTermManager.ts

```diff
@@ -84,10 +91,17 @@ export class MissingTermManager {
     const termIndex = this.termForcedToAppear.indexOf(term);
     this.termForcedToAppear.splice(termIndex, 1);
     this.queryStateModel.set('missingTerm', [...this.termForcedToAppear]);
+    this.logUAMissingTerm(term);
     this.queryController.executeQuery();
   }
 
   private handleClearBreadcrumb() {
     this.queryStateModel.set('missingTerm', []);
   }
+
+  private logUAMissingTerm(term: string) {
```

Comment:
Since the method is only logging remove events, it should be mentioned in the name.

```
private logMissingTermRemovedEvent(term: string) {}
```